### PR TITLE
Update documentation

### PR DESCRIPTION
**Pull request by ncaak**
This PR updates last gatherer's backported documentation with fixed releases from errata.

## Categories

- [ ] Bugfix
- [ ] Data Enhancement
- [ ] Feature
- [ ] Backporting
- [X] Others (CI, Infrastructure, Documentation)

## Sample Archive

- N/A

## Documentation

- N/A

## Unit Tests

- N/A

## Privacy

Yes. There are no sensitive data in the newly collected information.

## Changelog

Yes

## Breaking Changes

No

## References

https://issues.redhat.com/browse/CCXDEV-10420

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,14 +1,5 @@
...
-## 4.14
-
-### Data Enhancement
-- [#741](https://github.com/openshift/insights-operator/pull/685) Gather openshift-monitoring persistent volumes
-
-### Others
-
-- [#756](https://github.com/openshift/insights-operator/pull/756) add unit tests for cluster proxy gatherer
-
+	Please see OpenShift release notes for official changes\n<!--Latest hash: 08b4171784b8debe038e912ecd8bcab511cf7aa3-->
```
Comment (ncaak) [inline CHANGELOG.md:2]: Fixed changelog with the one generated from the script. I think those changes were done by me manually forever ago.

**Comment by tremes** [comment]
Versions update in the documentation.
/override ci/prow/e2e-agnostic-upgrade 
/lgtm

**Pull request merged**